src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  background-color: black;
+  margin: 0;
+}
+
+.wrap-block {
+  width: 350px;
+  margin: 20px 0 40px 20px;
+}
+
+.last-border {
+  border: 1px solid #e85450;
+}
+
+.third-border {
+  border: 10px solid #2c8ec1;
+}
+
+.second-border {
+  border: 1px solid #e85450;
+}
```

Comment:
is there any differences between `.second-border`, `.last-border` and border styles in `.text-box`?